probe/src/probemon.py

```diff
@@ -44,6 +50,11 @@ def __init__(self, size):
         self.vendor = LRU(size)
 
 def print_fields(fields):
+    global Count
+    global arr
+    if fields[1] not in arr:
```

Comment:
I'm inferring from this that"
* `fields[1]` is a MAC address
* `Count` is the number of MAC addresses seen thus far
* `arr` is an array of each MAC address seen

If this is the case, then we should:
* Replace `Count` with `len(arr)`, which returns the length of `arr`
* Remove the `global` declarations which should not be necessary in this context, since `arr` and `Count` are defined at the top-level - you would use `global` if you defined `arr` and `Count` inside `print_fields()` and wanted them to be accessible outside of that scope